Content/Items/Forest/Accessories.AcornSprout.cs

```diff
@@ -21,11 +21,24 @@ public override void SafeSetDefaults()
 		public override void Load()
 		{
 			StarlightPlayer.OnHitNPCWithProjEvent += SpawnAcorn;
+			StarlightPlayer.OnHitNPCEvent += SpawnAcorn2;
 		}
 
-		public override void Unload()
+		private void SpawnAcorn2(Player player, Item Item, NPC target, NPC.HitInfo hit, int damageDone)
```

Comment:
I personally dislike numbers in method names a lot. since this and SpawnAcorn have identical params. they could use another word each in the method name to differentiate them or least XML documentation strings to explain why they're the same but different